content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,294 @@
+to set it up locally.
+
+To then perform the model registry actions in this guide, follow
+[these steps](/doc/studio/user-guide/experiments/create-a-project#connect-to-a-git-repository-and-add-a-project)
+to sign in to DVC Studio, connect it to your GitHub account and add your forked
+repository as a DVC Studio project.
+
+</details>
+
+## Adding models
+
+Let's now train a model and add it to the model registry. We will be using
+[DVCLive](/doc/dvclive) and add a model using Python code. This will also
+automatically save the model to DVC.
+
+We use the [`log_artifact`](/doc/dvclive/live/log_artifact) method to
+<abbr>cache</abbr> the model with DVC and add it to the the model registry. Open
+the training script `src/train.py` in our example repository and have a look at
+the following code under the `with Live(...)` context:
+
+```python
+with Live(...) as live:
+
+...
+
+    live.log_artifact(
+        path="models/model.pkl",
+        type="model",
+        name="pool-segmentation",
+        desc="This is a Computer Vision (CV) model that's segmenting out swimming pools from satellite images.",
+        labels=["cv", "segmentation", "satellite-images"],
+    )
+```
+
+Here the `path` parameter tells DVC that our model is to be found under
+`"models/model.pkl"`, the `type` parameter is `"model"` and so it will show up
+in DVC Studio (other artifact types will not). The rest of the parameters are
+descriptive and optional and will also show up in DVC Studio.
+
+<details id="push-click-to-see-how-artifacts-are-registered">
+
+#### 💡 Expand to see how adding models to the registry works under the hood
+
+When we call the `log_artifact()` method, DVC takes all the information we
+provide in the call and edits the `dvc.yaml` file which will now contain the
+following lines:
+
+```yaml
+artifacts:
+  pool-segmentation:
+    path: models/model.pkl
+    type: model
+    desc:
+      This is a Computer Vision (CV) model that's segmenting out swimming pools
+      from satellite images.
+    labels:
+      - cv
+      - segmentation
+      - satellite-images
+```
+
+When you push the resulting file to your Git remote it is parsed by DVC Studio
+and it then shows all model artifacts from your `dvc.yaml` files in the model
+registry.
+
+</details>
+
+Now we just need to run the Python script which includes this code to cache and
+register the model. If you are
+[following](/docs/start/model-management/model-registry#follow-along-instructions)
+our example repository then this has already been done and we can continue to
+the next section.
+
+If you are building your own repository, you will need to run the script and
+push the result to your Git remote (GitHub) yourself. This is covered in detail
+in our
+[Experiment Management guide](/doc/start/experiments/experiment-pipelines).
+
+For a general introduction of how DVC stores and shares data, see our
+[Get Started guide on Data Versioning](/doc/start/data-management/data-versioning).
```

Comment:
Minor, but I would consider dropping all this text. People can probably figure out on their own to go to another get started section, and I don't see that this text adds much info besides linking to those sections.

Reply:
I would at least keep the first sentence though
```suggestion
If you are building your own repository, you will need to run the script and
push the result to your Git remote (GitHub) yourself. 
```